src/entities/committer.ts

```diff
@@ -35,20 +36,23 @@ export default class Committer {
 	_contract?: PoolCommitter;
 	address: string;
 	provider: ethers.providers.Provider | ethers.Signer | undefined;
+	multicallProvider: MCProvider.MulticallProvider | ethers.Signer | undefined;
```

Comment:
should be fine to type like so throughout:

`multicallProvider: MCProvider.MulticallProvider | undefined;`